src/cbmc/cbmc_parse_options.cpp

```diff
@@ -709,7 +726,11 @@ void cbmc_parse_optionst::preprocessing()
 }
 
 bool cbmc_parse_optionst::process_goto_program(
-  const optionst &options)
+  goto_modelt &goto_model,
+  const optionst &options,
+  const cmdlinet &cmdline,
```

Comment:
Your change highlights one existing poor design: this function should not be using `cmdline` anymore, anything it cares about ought to have been placed in `options`. Could you please clean this up as well?